library/src/blas2/rocblas_trsv_strided_batched.cpp

```diff
@@ -0,0 +1,273 @@
+                                                        rocblas_int       supplied_invA_size = 0,
+                                                        rocblas_stride    stride_invA        = 0)
+    {
+        if(!handle)
+            return rocblas_status_invalid_handle;
+
+        auto layer_mode = handle->layer_mode;
+        if(layer_mode & rocblas_layer_mode_log_trace)
+            log_trace(handle,
+                      rocblas_trsv_strided_batched_name<T>,
+                      uplo,
+                      transA,
+                      diag,
+                      m,
+                      A,
+                      lda,
+                      stride_A,
+                      B,
+                      incx,
+                      stride_B,
+                      batch_count);
+
+        if(layer_mode & (rocblas_layer_mode_log_bench | rocblas_layer_mode_log_profile))
+        {
+            auto uplo_letter   = rocblas_fill_letter(uplo);
+            auto transA_letter = rocblas_transpose_letter(transA);
+            auto diag_letter   = rocblas_diag_letter(diag);
+
+            if(layer_mode & rocblas_layer_mode_log_bench)
+            {
+                if(handle->pointer_mode == rocblas_pointer_mode_host)
+                    log_bench(handle,
+                              "./rocblas-bench -f trsv_strided_batched -r",
+                              rocblas_precision_string<T>,
+                              "--uplo",
+                              uplo_letter,
+                              "--transposeA",
+                              transA_letter,
+                              "--diag",
+                              diag_letter,
+                              "-m",
+                              m,
+                              "--lda",
+                              lda,
+                              "--stride_A",
+                              stride_A,
+                              "--incx",
+                              incx,
+                              "--stride_B",
+                              stride_B,
+                              "--batch",
+                              batch_count);
+            }
+
+            if(layer_mode & rocblas_layer_mode_log_profile)
+                log_profile(handle,
+                            rocblas_trsv_strided_batched_name<T>,
+                            "uplo",
+                            uplo_letter,
+                            "transA",
+                            transA_letter,
+                            "diag",
+                            diag_letter,
+                            "M",
+                            m,
+                            "lda",
+                            lda,
+                            "stride_A",
+                            stride_A,
+                            "incx",
+                            incx,
+                            "stride_B",
```

Comment:
In our testing code we're using stride_x I believe, rather than stride_B. stride_x makes more sense since it's a vector not a matrix imo.